src/sentry/static/sentry/app/views/projectGeneralSettings.jsx

```diff
@@ -184,235 +176,69 @@ export default class ProjectGeneralSettings extends AsyncView {
-                    Referer: <code>Referer</code>,
-                  }
-                )}
-              </p>
-              <ListAsTextareaField
-                name="allowedDomains"
-                label={t('Allowed domains')}
-                help={t('Separate multiple entries with a newline. Cannot be empty.')}
-                placeholder={t('e.g. https://example.com or example.com')}
-              />
-              <BooleanField
-                name="scrapeJavaScript"
-                label={t('Enable JavaScript source fetching')}
-                help={t(
-                  'Allow Sentry to scrape missing JavaScript source context when possible.'
-                )}
-              />
-
-              <DynamicWrapper
-                value={
-                  <TextField
-                    name="securityToken"
-                    label={t('Security token')}
-                    help={t(
-                      'Outbound requests matching Allowed Domains will have the header "{token_header}: {token}" appended.'
-                    )}
-                  />
-                }
-                fixed={
-                  <TextField
-                    name="percy-mock-securityToken"
-                    label={t('Security token')}
-                    value="<<SECURITY_TOKEN>>"
-                    help={t(
-                      'Outbound requests matching Allowed Domains will have the header "{token_header}: {token}" appended.'
-                    )}
-                  />
-                }
-              />
-
-              <TextField
-                name="securityTokenHeader"
-                label={t('Security token header')}
-                help={t(
-                  'Outbound requests matching Allowed Domains will have the header "{token_header}: {token}" appended.'
-                )}
-                placeholder={t('e.g. X-Sentry-Token')}
-              />
-              <BooleanField
-                name="verifySSL"
-                label={t('Verify TLS/SSL')}
-                help={t(
-                  'Outbound requests will verify TLS (sometimes known as SSL) connections.'
-                )}
-              />
-            </div>
-          </div>
-          <div className="box">
-            <div className="box-header">
-              <h3>{t('Remove Project')}</h3>
-            </div>
-            <div className="box-content with-padding">{this.renderRemoveProject()}</div>
-          </div>
-          <div className="box">
-            <div className="box-header">
-              <h3>{t('Transfer Project')}</h3>
-            </div>
-            <div className="box-content with-padding">{this.renderTransferProject()}</div>
-          </div>
-        </ApiForm>
+          <JsonForm
+            forms={projectFields}
+            additionalFieldProps={{organization}}
+            access={new Set(organization.access)}
+            renderBodyStart={({title}) => {
+              if (title === 'Client Security') {
```

Comment:
What would you think of putting this in the form data instead? Not sure how doable it is but comparing on these title strings seems non-ideal

Reply:
Agree this isn't good, but I'm going to leave it for now because this may not be used that frequently. I think instead of sticking it into form data, we'll want to decouple fields from the form group in form data.